Makefile

```diff
@@ -201,7 +201,7 @@ docker-integration-tests:
 
 $(tether-linux): $(shell find cmd/tether -name '*.go') metadata/*.go
 	@echo building tether-linux
-	@CGO_ENABLED=1 GOOS=linux GOARCH=amd64 $(GO) build $(RACE) -tags netgo -installsuffix netgo --ldflags '-extldflags "-static"' -o ./$@ ./cmd/tether
+	@GOOS=linux GOARCH=amd64 $(GO) build $(RACE) -tags netgo -installsuffix netgo --ldflags '-extldflags "-static"' -o ./$@ ./cmd/tether
```

Comment:
I think CGO was explicitly enabled in be658a65a0675daa3bcbe0b2f7ca88bcf5a7d056 to ensure we had a static tether binary - does this still produce a static binary?